scripts/check-all-versions.sh

```diff
@@ -0,0 +1,92 @@
+#! /bin/bash
+set -eo pipefail
+
+RED='\033[0;31m'
+NO_COLOR='\033[0m'
+
+if mkdir tmp_repos; then
+  cd tmp_repos
+  git clone https://github.com/FairwindsOps/charts
+  git clone https://github.com/FairwindsOps/insights-plugins
+  git clone https://github.com/FairwindsOps/polaris
+  git clone https://github.com/FairwindsOps/goldilocks
+  git clone https://github.com/FairwindsOps/pluto
+  git clone https://github.com/FairwindsOps/nova
+  cd ..
+fi
+cd tmp_repos
+
+values_file="./charts/stable/insights-agent/values.yaml"
+
+declare -A latest_versions=()
+declare -A used_versions=()
+
+cloned_projects=( polaris goldilocks nova pluto )
+
+for proj in ${cloned_projects[@]}; do
+  cd $proj
+  latest_versions[$proj]=$(git describe --tags --abbrev=0)
+  cd ..
+  used_versions[$proj]=$(yq e ".$proj.image.tag" $values_file)
+done
+
+plugin_projects=( aws-costs falco-agent kube-bench kube-bench-aggregator opa prometheus rbac-reporter right-sizer trivy uploader workloads )
+declare -A rewrites=()
+for proj in ${plugin_projects[@]}; do
+  rewrites[$proj]=$proj
+done
+rewrites["aws-costs"]="awscosts"
+rewrites["falco-agent"]="falco"
+rewrites["kube-bench-aggregator"]="kube-bench.aggregator"
+rewrites["prometheus"]="prometheus-metrics"
+
+for proj in ${plugin_projects[@]}; do
+  latest_versions[$proj]=$(cat ../plugins/$proj/version.txt)
```

Comment:
Does the insights-plugins repo need to be re-cloned? It looks like that copy is not used.